libmscore/instrchange.cpp

```diff
@@ -256,6 +272,17 @@ void InstrumentChange::read(XmlReader& e)
                   _instrument->read(e, part());
             else if (tag == "init")
                   _init = e.readBool();
+            else if (tag == "staffGroup") {
+                  QString text = e.readElementText();
+                  if (text == "percussion")
+                        _staffGroup = StaffGroup::PERCUSSION;
+                  else if (text == "tab")
+                        _staffGroup = StaffGroup::TAB;
+                  else
+                        _staffGroup = StaffGroup::STANDARD;
```

Comment:
Is `_staffGroup` set to `StaffGroup::STANDARD` by default in `instrChange.h`? Otherwise, it will technically be uninitialized on read.